packages/react-scripts/scripts/build.js

```diff
@@ -141,3 +142,30 @@ function copyPublicFolder() {
     filter: file => file !== paths.appHtml,
   });
 }
+
+function formatError(err) {
+  const message = get(err, 'message');
+
+  // Add more helpful message for UglifyJs error
+  if (typeof message === 'string' && message.indexOf('from UglifyJs') !== -1) {
+    try {
+      console.log(
+        'UglifyJs could not parse the code from \n\n',
+        chalk.yellow(
+          err.stack.split('\n')[1].split('[')[1].split('][')[0].replace(']', '')
```

Comment:
You can use https://regex101.com/ for testing regexes.

I don't mind split/joining too much either, but then we need to test that it matches the format first somehow. Before we attempt to parse. So that we don't end up with garbage.